### DISCUSSION TITLE
[Feature Request] Offset Adjustment on Score Screen

### DISCUSSION THREAD

**Discussion by XeroOl** [TeamRizu/OutFox#230] [03 - Feature Requests]
Sometimes, I need to adjust my global offset, or a song offset, or even a song's bpm. Usually, this involves trial and error, where I adjust the desired setting, play the song, and then readjust.

**Idea / Solution:**
On the score screen, the game shows a variety of helpful timing data (timing plot, score, etc).  It would be cool if this timing data would retroactively adjust when changing the global offset / song offset / song bpm. This way, I could just play a song once, and then I could choose an offset based on the timing data, without needing to replay the song after each adjustment to the offset. That way, I can better evaluate how well a particular offset fits, and make sure files that I create are synced perfectly.

**Comment by Scraticus** [comment]
This is in progress, good suggestion!
